packages/elcollectooorr/agents/elcollectooorr/tests/helpers/constants.py

```diff
@@ -24,11 +24,9 @@
 
 
 CUR_PATH = os.path.dirname(inspect.getfile(inspect.currentframe()))  # type: ignore
-ROOT_DIR = Path(CUR_PATH, "..", "..").resolve().absolute()
-TEST_DATA_DIR = ROOT_DIR / "tests" / "data"
-CONTRACTS_PACKAGES_DIR = ROOT_DIR / "packages" / "valory" / "contracts"
-ELCOL_CONTRACT_PACKAGES = ROOT_DIR / "packages" / "elcollectooorr" / "contracts"
-THIRD_PARTY = ROOT_DIR / "third_party"
+ROOT_DIR = Path(CUR_PATH, "..", "..", "..", "..",).resolve().absolute()
+TEST_DATA_DIR = ROOT_DIR / "agents" / "elcollectooorr" / "tests" / "data"
+ELCOL_CONTRACT_PACKAGES = ROOT_DIR / "contracts"
```

Comment:
Just import from the package: `packages.author.contracts....`

Reply:
The path is needed, the files need to be read in `artblocks_utils.py`.

Reply:
Yes but the way you do it is messy. In the target folder, introduce a constant like `PACKAGE_DIR = Path(__file__).parent.parent` for instance. Then here do `from ... import PACKAGE_DIR`